tests/data/expected/main/jsonschema/serialized_decimal_defaults/msgspec_Struct.py

```
@@ -0,0 +1,15 @@
+
+from __future__ import annotations
+
+from decimal import Decimal
+
+from msgspec import UNSET, Struct, UnsetType
+
+
+class Invoice(Struct):
+    amount: Decimal | UnsetType = Decimal('12.340')
+    invalid_amount: Decimal | UnsetType = 'not-a-decimal'
+    nullable_amount: Decimal | None | UnsetType = UNSET
```

Ruff (0.16.2)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)